TransIT.API/Controllers/BillController.cs

```diff
@@ -27,109 +27,74 @@ public BillController(IServiceFactory serviceFactory, IFilterServiceFactory filt
         [HttpGet]
         public async Task<IActionResult> Get([FromQuery] uint offset = 0, uint amount = 1000)
         {
-            try
+            var result = await _billService.GetRangeAsync(offset, amount);
+            if (result != null)
             {
-                var result = await _billService.GetRangeAsync(offset, amount);
-                if (result != null)
-                {
-                    return Json(result);
-                }
-                else
-                {
-                    return null;
-                }
+                return Json(result);
             }
-            catch (Exception e)
+            else
```

Comment:
why you can't return just Json(result);, without any 'if else'?

Reply:
Will fix it